js/focuspoint/focuspoint.js

```diff
@@ -197,10 +262,10 @@
 			{
 				var $cropButton = $('.' + css.imageFocus.button);
 
-				if(data.success === false){
+				if (data.success === false) {
 					Plugin.prototype.activateCropButton();
 					$cropButton.text('Please try again');
-				}else{
+				} else {
```

Comment:
I have updated this in the following feature: feature/improve-code-and-functionality